package.json

```diff
@@ -5,16 +5,17 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "./scripts/build.sh",
+    "build": "ng-packagr -p packages/nx/ng-package.json && ./scripts/build.sh",
     "e2e": "./scripts/e2e.sh",
     "format": "precise-commits --whitelist=\"{packages,e2e}/**/*.ts\"",
     "linknpm": "./scripts/link.sh",
     "package": "./scripts/package.sh",
     "release": "./scripts/release.sh",
     "copy": "./scripts/copy.sh",
     "test:schematics": "yarn linknpm && ./scripts/test_schematics.sh",
-    "test:nx": "yarn linknpm && ./scripts/test_nx.sh",
-    "test": "yarn linknpm && ./scripts/test_nx.sh && ./scripts/test_schematics.sh",
+    "test:nx": "./scripts/link-spec.sh && ./scripts/test_nx.sh",
```

Comment:
I'm not sure why `link-spec` is required given that test:schematics still invokes `linknpm`.

Reply:
Its now been removed, I combined the two scripts. The issue we had is that ngpackagr takes an entry point to determine which files to package. The entry point did not point to any spec files so we needed a different building mechanism to transpile our spec.ts files